src/colormaps.jl

```diff
@@ -39,26 +39,26 @@ function show(io::IO, c::ColorMap)
 end
 
 # all Python dependencies must be initialized at runtime (not when precompiled)
-const colorsm = PyNULL()
+const mcolors = PyNULL()
 const cm = PyNULL()
 const LinearSegmentedColormap = PyNULL()
 const cm_get_cmap = PyNULL()
 const cm_register_cmap = PyNULL()
 const ScalarMappable = PyNULL()
 const Normalize01 = PyNULL()
 function init_colormaps()
-    copy!(colorsm, pyimport("matplotlib.colors"))
+    copy!(mcolors, pyimport("matplotlib.colors"))
```

Comment:
Why?

Reply:
Because `matplotlib` is the prefix, thus I think it should be before rather than after the word. However, simply `colors` would work better, but I suppose that was not chosen for some reason.

Reply:
We shouldn't change it gratuitiously since it has been around for awhile — some code might be using it, even though it isn't exported.   `colorsm` is fine, I would just leave it as is.

(I thins `colorsm` originally stood for "colors module".)